src/app/imex/sync/check-for-update.util.ts

```diff
@@ -38,7 +38,9 @@ export const checkForUpdate = (params: { remote: number, local: number, lastSync
     } else if (lastSync < local) {
       return UpdateCheckResult.RemoteUpdateRequired;
     } else if (lastSync === local) {
-      alert('Sync Warning: Dropbox date not up to date despite seemingly successful sync. (This might happen when: 1. You have conflict changes and decide to take the local version. 2. You open the other instance and also decide to use the local version.)');
+      alert('Sync Warning: Dropbox date not up to date despite seemingly successful sync.\
```

Comment:
I also would rather ignore this line.

Reply:
Should be ok now, I disabled the max-length for strings